Update release-overview.md (#1062)
mattermost/mattermost-handbook: operations/research-and-development/product/release-process/release-overview.md

For P0 bugs (eg. bugs that affect more than 25% of our customers with system degradation), we will do an exception and rollout a hotfix as soon as possible.

**Schedule for Cloud releases**:

>  - Thursdays: Release day. Also merge master into the cloud branch and update cloud test servers, including Rainforest RFQA-Cloud servers.

Reviewer: ```suggestion
 - Thursdays: Release day. Also merge master into the cloud branch and update cloud test servers for the next release, including Rainforest RFQA-Cloud servers.
```

Author: Oops! Sorry Amy, I think I messed up and committed your suggestion in error? Apologies - I don't know how to undo it?

Reviewer: You did it correctly :) Normally the process is for the PR submitter to commit reviewer suggestions.

Author: Aah, pshew! :)  Thank goodness...  Been a while so I'm a bit rusty on PR processes... Thanks Amy

>  - Fridays: Run Rainforest release test run groups, Cypress automated tests and product high level release smoke tests (i.e. Boards, Calls, Channels, Integration Frameworks, Playbooks and Suite Users)

Reviewer 2: ```suggestion
 - Fridays: Run Rainforest release test run groups, Cypress automated tests and product high level release smoke tests (i.e. Boards, Calls, Channels, Integration Frameworks, Playbooks and Suite Users). Close or label tickets for the release.
```

>  - Mondays and Tuesdays: Test failure reviews and bug fixing.

Reviewer 2: ```suggestion
 - Mondays and Tuesdays: Test failure reviews and bug fixing. Close or label any remaining tickets. Each team signs off as appropriate.
```

 - Wednesdays: Final QA tasks and sign-off.

>     - QA approval should be given by the EOD Wednesday so that the release rollout can be started early Thursday morning. 

Reviewer 2: ```suggestion
    - QA approval should be given by 5pm Eastern on Wednesday so that the release rollout can be started early Thursday morning. 
```
